proxy/http2/HPACK.cc

```diff
@@ -332,18 +332,41 @@ int64_t
 encode_string(uint8_t *buf_start, const uint8_t *buf_end, const char *value, size_t value_len)
 {
   uint8_t *p = buf_start;
+  bool use_huffman = true;
```

Comment:
I left the flag for further use. There are some cases that plain text is shorter than huffman encoded data, and the codes for both cases would be used when we found a smart way to switch the encoding.